rosdep/python.yaml

```diff
@@ -1628,6 +1629,7 @@ python-gnupg:
   arch: [python2-gnupg]
   debian: [python-gnupg]
   fedora: [python-gnupg]
+  freebsd: [py27-gnupg]
```

Comment:
I didn't find a package with this name. Did you mean [py27-python-gnupg](https://www.freshports.org/security/py-python-gnupg/)?

Reply:
Yes, I think I did. It's strange because I know I built melodic-desktop in a jail, which meant that rosdep would have called pkg and pkg would fail if it gets a bad package name. I'll investigate and let you know what I find (with a possible change).

Reply:
OK. I did a bit more digging. The origin of the py27-gnupg pkg is [security/py-gnupg](https://www.freshports.org/security/py-gnupg/) on FreeBSD. It seems to be a fork of the python-gnupg with an aim on security. It works, but you need to change the python import statement.

`>>> from pretty_bad_protocol import gnupg`

I've changed python.yaml to use py27-python-gnupg as it doesn't require changing other files in ROS.

Thanks for checking this out.